source/php/Helper/SingleEventData.php

```diff
@@ -111,9 +111,28 @@ public static function getFormattedTimeDiff($start, $end)
         // Add one day to number of days if the diff is more than 24 hours but less than 1 week
         return $diff >= 24 && $diff < (24 * 7)
             ? sprintf(__('%d days', 'event-integration'), (floor($diff / 24)) + 1)
-            : human_time_diff($start, $end);
+            : self::eventOccasionDuration($start, $end);
     }
 
+    // Display hours + minutes if event duration is between 1 hour and 1 day, 
```

Comment:
Explain why this is needed. human_time_diff should be enough: 
https://core.trac.wordpress.org/browser/tags/6.1/src/wp-includes/formatting.php#L3788

Why isen't it?